package.json

```diff
@@ -34,5 +37,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^6.1.0",
```

Comment:
downgrade this to 5.6.0 please, it will run better with Semaphore 👍🏻

Reply:
This is now fixed!